helpers/voting/formatVotes.ts

```diff
@@ -121,5 +121,7 @@ export function formatVoteStringWithPrecision(
   const identifierPrecision = BigNumber.from(
     getPrecisionForIdentifier(decodedIdentifier)
   ).toString();
-  return formatFixed(vote, identifierPrecision).toString();
+  const formatted = formatFixed(vote, identifierPrecision).toString();
+  // if the formatted number ends with .0, remove the .0
```

Comment:
Ethers formats whole numbers with decimal points like "1.0" which breaks the matching logic. Remove that redundant decimal if present.